lint.sh

```diff
@@ -16,6 +16,7 @@ MATCHES=$(grep -ni 'xxx' "$1" | perl -lpe 'print "\nPossible incomplete sections
   grep -niE '\s+$' "$1" | perl -lpe 'print "\nTrailing whitespace:" if $. == 1'
   perl -ne '$/ = "\n\n"; print "$_" if (/class="?(note|example).+(\n.+)*\s+(should|must|may|optional|recommended)(\s|$)/mi)' "$1" | perl -lpe 'print "\nRFC2119 keyword in example or note (use: might, can, has to, or override with <!--non-normative-->must):" if $. == 1'
   perl -ne '$line++; $in_domintro = 1 if (/^  <dl class="domintro">$/); print "$line: $_" if ($in_domintro && /\s+(should|must|may|optional|recommended)(\s|$)/i); $in_domintro = 0 if (/^  <\/dl>$/)' "$1" | perl -lpe 'print "\nRFC2119 keyword in domintro (use: might, can, has to, or override with <!--non-normative-->must):" if $. == 1'
+  grep -ni 'class="idl"' "$1" | grep -vF '<code class="idl" data-x="">' | perl -lpe 'print "\nIrregular use of class=\"idl\":" if $. == 1'
```

Comment:
```
grep -ni 'class="idl"' "$1" | grep -vF '<code class="idl" data-x="">'
```
The second grep there should handle the case of `code` elements without `data-x=""` — because wattsi no longer requires it for `code` elements that are children of `pre` elements. See https://github.com/whatwg/wattsi/commit/51e5c2df3d5a17632f6d44f00032f2e801657c87

And https://github.com/whatwg/html/pull/3948 will remove all the `data-x=""` that were added to `code`-element children of `pre` (when the change was made that moved the `class` from the `pre` elements to `code` children`.

Also, that second grep would fail for any case where there’s an extra space before `class` or if another attribute has been added to the `code` element before the `class` attribute (which I don’t see at this point we’d ever have any need to be doing, but who knows).

So something more like the following would be more robust — 

```
grep -ni 'class="idl"' "$1" | grep  '<code[^>]\+class="idl"' 
```